app/process/watcher/consensus-message/consensus-message.go

```diff
@@ -1,3 +1,53 @@
 package consensus_message
 
-// todo consensus message watcher
+import (
+	"encoding/json"
+	hederasdk "github.com/hashgraph/hedera-sdk-go"
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+	"github.com/limechain/hedera-watcher-sdk/queue"
+	"github.com/limechain/hedera-watcher-sdk/types"
+	"log"
+)
+
+type ConsensusTopicWatcher struct {
```

Comment:
Add NewConsensusTopicWatcher, which takes a HederaMirrorClient, topicId and typeMessage (to which the watcher will push messages) as parameters and returns *ConsensusTopicWater